CHANGELOG.md

```diff
@@ -16,6 +16,9 @@
   `reg u32 x = 1, y, z = 2; stack u64 s = 1, u, t = z;`
   ([PR #908](https://github.com/jasmin-lang/jasmin/pull/908)).
 
+- The SCT checker now allows protecting in MMX registers
+  ([PR #917](https://github.com/jasmin-lang/jasmin/pull/917)).
+
```

Comment:
It might be worth mentioning the availability of the MMX instruction POR.

Reply:
Good idea